release_app.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
```

Comment:
nice! works like charm, but its only executable via a bash shell. windows users may be unable to run this with ease. could we revise this to accommodate all OSs?

Reply:
we may need to use a cross-platform scripting language like python or create separate scripts for each platform. kinda sucks though. maybe there's another way i don't know of?